ONSdigital/ssdc-rm-docker-dev
# 3420 Improve setup pubsub script

# Motivation and Context

The setup pubsub script had got very noisy and long, refactor to make it quiet unless something breaks, and easier to read and modify the topics and subscriptions it's setting up. Also some small house keeping, formatting etc.

# What has changed



- Rewrite setup pubsub script
- Update python dependencies
- Move python message tools into module
- Tidy the makefile
- Format README

# How to test?



Check pubsub is still being setup correctly (AT's still pass etc.). Check the message tools still function the same.

# Links



https://trello.com/c/L6NHDRux/3420-improve-setup-pubsub-script-in-docker-dev
